basics_of_sst-core/Example00/ExampleComponent.h

```diff
@@ -0,0 +1,76 @@
+#ifndef MY_COMPONENT_H
+#define MY_COMPONENT_H
+
+#include <sst/core/component.h>
+#include <sst/core/output.h>
+#include <sst/core/params.h>
+
+namespace Example00
+{
+    // Define the log levels.  Each level displays its own message and the
+    // levels above it.
+    //
+    // For example, ALL displays all log messages, INFO displays INFO, WARN,
+    // and FATAL messages, FATAL displays only FATAL messages.
+    //
+    const uint64_t FATAL = 0;
+    const uint64_t WARN  = 1;
+    const uint64_t INFO  = 2;
+    const uint64_t DEBUG = 3;
+    const uint64_t TRACE = 4;
+    const uint64_t ALL   = 5;
+
+    // This is a very simple component.  It only registers a clock and 
+    // prints log messages as the clock handler is called.
+    //
+    // Remember, all components inherit from SST::Component
+    //
+    class ExampleComponent : public SST::Component
+    {
+        public:
+            // Constructor/Destructor
+            //
+            ExampleComponent(SST::ComponentId_t id, SST::Params &params);
+            ~ExampleComponent() {}
+
+            // Standard SST::Component functions.  These all need to
+            // be implemented in the component, even if they are empty.
+            //
```

Comment:
These do not need to be implemented by components if they are empty. Change the comment?